lib/util/stream_utils.js

```diff
@@ -426,6 +429,13 @@ shaka.util.StreamUtils = class {
 
       const audio = variant.audio;
 
+      if (isFirefoxAndroid && audio && audio.encrypted &&
```

Comment:
I would add a comment explaining why we are filtering this. Maybe referencing #6111?

Reply:
Done!